jmclient/jmclient/configure.py

```diff
@@ -164,6 +164,24 @@ def jm_single():
 ##usessl = false
 ##socks5 = true
 
+[MESSAGING:server4]
+host = irc.oftc.net
+## force IPv4 only
+# host = irc4.oftc.net
+## force IPv6 only
+# host = irc6.oftc.net
```

Comment:
Haven't done any testing yet, but what could be the reason for anybody to want to force IPv4 or IPv6? With JoinMarket we anyway prefer Tor or possibly other privacy networks in the future.

Reply:
The default address supports both in the dual stack, therefore IPv4 and IPv6. 

Of course, it matters less for accessibility, because tor translates completely transparently from IPv4 or IPv6 client to IPv4 or IPv6 host and vice versa. 


The difference lies in how the client is configured. The tor socksport included in the tor browser sets IPv6Traffic PreferIPv6 by default socks_port_flags. 

While the tor client daemon does not.

So you could make a difference between both clients.

Just added it to give the user maximum control, feel free to remove the comment in config if it seems unnecessary.




Re: 

> other privacy networks 

agora irc already supports i2p